### PR TITLE
chore(release): 1.2.1

### PR DESCRIPTION
**Pull request by erkarl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,75 @@
+## [1.2.1](https://github.com/ExchangeUnion/xud/compare/v1.0.0...v1.2.1) (2020-11-19)
+
+
```
Comment (kilrau) [inline CHANGELOG.md:3]:
```suggestion
This maintenance release includes a number of features, most significantly it switches xud to use lnd's new `sendpaymentv2` call to prepare for multi-channel trades. It also improves xud's shutdown behavior and fixes several bugs.```

**Review by kilrau — CHANGES_REQUESTED** [review]
Also Bug Fixes and Features contain duplicates from https://github.com/ExchangeUnion/xud/releases/tag/v1.2.0

**Review by sangaman — APPROVED** [review]
Looks good but I think a brief write up like kilian proposed above would be good.

**Comment by erkarl** [comment]
Updated.

**Comment by kilrau** [comment]
Still need to remove duplicates from `Bug Fixes` and `Features`.

**Comment by erkarl** [comment]
> Still need to remove duplicates from Bug Fixes and Features.

Updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,74 @@
+## [1.2.1](https://github.com/ExchangeUnion/xud/compare/v1.0.0...v1.2.1) (2020-11-19)
+
+This maintenance release includes a number of features, most significantly it switches xud to use lnd's new `sendpaymentv2` call to prepare for multi-channel trades. It also improves xud's shutdown behavior and fixes several bugs.
+
+### Bug Fixes
+
+* added swap success log entry ([#1649](https://github.com/ExchangeUnion/xud/issues/1649)) ([e914ec0](https://github.com/ExchangeUnion/xud/commit/e914ec028f6420bbf2d973b92626f6284a9a8ba5))
+* graceful shutdown ([#1994](https://github.com/ExchangeUnion/xud/issues/1994)) ([cde3203](https://github.com/ExchangeUnion/xud/commit/cde320312dcd08f70123b5949d038205748d6148))
+* **cli:** switch getbalance table columns ([#1931](https://github.com/ExchangeUnion/xud/issues/1931)) ([49c8c18](https://github.com/ExchangeUnion/xud/commit/49c8c18cf9d00ede7f84ce617921ffc3aa9b18de)), closes [#1930](https://github.com/ExchangeUnion/xud/issues/1930)
```
Comment (kilrau) [inline CHANGELOG.md:9]:
```suggestion

```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,74 @@
...
+* alias missing in streamorders ([#1725](https://github.com/ExchangeUnion/xud/issues/1725)) ([#1962](https://github.com/ExchangeUnion/xud/issues/1962)) ([d537206](https://github.com/ExchangeUnion/xud/commit/d537206abec4835f96807853c66b23acaeb62e88))
+* **cli:** openchannel assertion error for string amount ([#1950](https://github.com/ExchangeUnion/xud/issues/1950)) ([a0cc6d7](https://github.com/ExchangeUnion/xud/commit/a0cc6d74dc73176cb43ba496a73f32be31adb8cd)), closes [#1643](https://github.com/ExchangeUnion/xud/issues/1643)
+* **connext:** not enough balance for closechannel ([#1963](https://github.com/ExchangeUnion/xud/issues/1963)) ([6599d88](https://github.com/ExchangeUnion/xud/commit/6599d885cfce3361715b99037013aca89220a658))
+* **orderbook:** don't remove 0 quantity w/ hold ([#1921](https://github.com/ExchangeUnion/xud/issues/1921)) ([5f3793a](https://github.com/ExchangeUnion/xud/commit/5f3793a0dc5d4c1cd042a2b5260d3db1942de750))
```
Comment (kilrau) [inline CHANGELOG.md:19]:
```suggestion

```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,74 @@
...
+* **p2p:** don't reconnect peers when pool closed ([#1965](https://github.com/ExchangeUnion/xud/issues/1965)) ([8483e1e](https://github.com/ExchangeUnion/xud/commit/8483e1e35d7fdc80404cac59768ddc61a773b465)), closes [#1668](https://github.com/ExchangeUnion/xud/issues/1668) [/github.com/ExchangeUnion/xud/issues/1668#issuecomment-684828808](https://github.com//github.com/ExchangeUnion/xud/issues/1668/issues/issuecomment-684828808)
+* **rpc:** no success if no channels to close ([#1689](https://github.com/ExchangeUnion/xud/issues/1689)) ([#1942](https://github.com/ExchangeUnion/xud/issues/1942)) ([17e40de](https://github.com/ExchangeUnion/xud/commit/17e40deda3ced6de23030094b6ada5f11dd383df))
+* bold link, mm link ([#1873](https://github.com/ExchangeUnion/xud/issues/1873)) ([54bd9ce](https://github.com/ExchangeUnion/xud/commit/54bd9ce11fc6297319b4c48c7fe9537b3031817b))
```
Comment (kilrau) [inline CHANGELOG.md:22]:
```suggestion

```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,74 @@
...
+* checking for invoice support on lnd clients while verifying connection ([4c26aff](https://github.com/ExchangeUnion/xud/commit/4c26aff0bafc52053aaf928a4d1936bab4293950))
```
Comment (kilrau) [inline CHANGELOG.md:23]:
```suggestion

```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,74 @@
...
+* don't activate unsupported pairs with peers ([0fe2d66](https://github.com/ExchangeUnion/xud/commit/0fe2d66d85c2fcc2d559740662977750ddae21e1))
```
Comment (kilrau) [inline CHANGELOG.md:24]:
```suggestion

```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,74 @@
...
+* getinfo output for lnd channels ([#1940](https://github.com/ExchangeUnion/xud/issues/1940)) ([#1948](https://github.com/ExchangeUnion/xud/issues/1948)) ([962daa4](https://github.com/ExchangeUnion/xud/commit/962daa46081bc3122a04afa82bbe88eb53ec0026))
+* grpc throws error for addpair/withdraw for wrong argument ([#1844](https://github.com/ExchangeUnion/xud/issues/1844)) ([48a0a33](https://github.com/ExchangeUnion/xud/commit/48a0a33811c2ba52f3791b73f1eef8b8933e09cc))
```
Comment (kilrau) [inline CHANGELOG.md:26]:
```suggestion

```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,74 @@
...
+* handling insufficient balance errors for swap clients ([246889b](https://github.com/ExchangeUnion/xud/commit/246889b89f7f9fa27ba47a31d1a2408d7a4a58a7))
```
Comment (kilrau) [inline CHANGELOG.md:27]:
```suggestion

```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,74 @@
...
+* implemented showing all pairs instead of active ones for listpeers ([206dad0](https://github.com/ExchangeUnion/xud/commit/206dad04b0151e3514f7883e066c361c034defca))
```
Comment (kilrau) [inline CHANGELOG.md:28]:
```suggestion

```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,74 @@
...
+* listorders limit displays first orders instead of last ([#1883](https://github.com/ExchangeUnion/xud/issues/1883)) ([e101e6f](https://github.com/ExchangeUnion/xud/commit/e101e6fb3d0ede2cf6439d10dafa410b533035da))
```
Comment (kilrau) [inline CHANGELOG.md:29]:
```suggestion

```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,74 @@
...
+* manual ban causes an undefined log entry ([#1779](https://github.com/ExchangeUnion/xud/issues/1779)) ([e7f8178](https://github.com/ExchangeUnion/xud/commit/e7f817841fccee30415f754816644f161fe6cc2c))
+* order invalidation only be sent to peers with active pair ([#1530](https://github.com/ExchangeUnion/xud/issues/1530)) ([#1890](https://github.com/ExchangeUnion/xud/issues/1890)) ([0dc85bd](https://github.com/ExchangeUnion/xud/commit/0dc85bd83ad0514241e85ae3a958bb01d31594b0))
+* propagating currencies and pairs on database if initDB true even DB is filled ([17c7cb5](https://github.com/ExchangeUnion/xud/commit/17c7cb54c3a55189ca459a9a842a7e2aa4886a6b))
```
Comment (kilrau) [inline CHANGELOG.md:32]:
```suggestion

```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,74 @@
...
+* propagating nodes on database if initDB true even DB is filled ([#1907](https://github.com/ExchangeUnion/xud/issues/1907)) ([246136e](https://github.com/ExchangeUnion/xud/commit/246136e3d2992711ce5e09d132cf79348bf99e6e))
```
Comment (kilrau) [inline CHANGELOG.md:33]:
```suggestion

```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,74 @@
...
+* rename tradinglimits reserved_inbound json_name ([7e56011](https://github.com/ExchangeUnion/xud/commit/7e560111dbb10079a6e9c488d46fb0ecc8bb1cb6))
+* xudrpc GetBalanceResponse json_name ([#1909](https://github.com/ExchangeUnion/xud/issues/1909)) ([26f89e6](https://github.com/ExchangeUnion/xud/commit/26f89e6d01cce2ec9d66dc39c0538d142a5af18c))
```
Comment (kilrau) [inline CHANGELOG.md:35]:
```suggestion

```

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,3 +1,74 @@
...
+* **connext:** avoid scientific notation for amount ([#1905](https://github.com/ExchangeUnion/xud/issues/1905)) ([cc1e689](https://github.com/ExchangeUnion/xud/commit/cc1e689181bfe4bafa19112d634a6eb12178cbe9))
```
Comment (kilrau) [inline CHANGELOG.md:36]:
```suggestion

```

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,3 +1,74 @@
...
+* **connext:** remove BigInt to avoid precision loss ([#1893](https://github.com/ExchangeUnion/xud/issues/1893)) ([d9ddd1c](https://github.com/ExchangeUnion/xud/commit/d9ddd1cfa16614fc914c61ded2c5e84fd7bd743c))
```
Comment (kilrau) [inline CHANGELOG.md:37]:
```suggestion

```

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,3 +1,74 @@
...
+* handling insufficient balance errors for swap clients ([246889b](https://github.com/ExchangeUnion/xud/commit/246889b89f7f9fa27ba47a31d1a2408d7a4a58a7))
+* implemented showing all pairs instead of active ones for listpeers ([206dad0](https://github.com/ExchangeUnion/xud/commit/206dad04b0151e3514f7883e066c361c034defca))
+* listorders limit displays first orders instead of last ([#1883](https://github.com/ExchangeUnion/xud/issues/1883)) ([e101e6f](https://github.com/ExchangeUnion/xud/commit/e101e6fb3d0ede2cf6439d10dafa410b533035da))
+* manual ban causes an undefined log entry ([#1779](https://github.com/ExchangeUnion/xud/issues/1779)) ([e7f8178](https://github.com/ExchangeUnion/xud/commit/e7f817841fccee30415f754816644f161fe6cc2c))
+* order invalidation only be sent to peers with active pair ([#1530](https://github.com/ExchangeUnion/xud/issues/1530)) ([#1890](https://github.com/ExchangeUnion/xud/issues/1890)) ([0dc85bd](https://github.com/ExchangeUnion/xud/commit/0dc85bd83ad0514241e85ae3a958bb01d31594b0))
```
Comment (kilrau) [inline CHANGELOG.md:31]:
```suggestion

```

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,3 +1,74 @@
...
+* removeorder output changed to a more meaningful message ([#1526](https://github.com/ExchangeUnion/xud/issues/1526)) ([ba04193](https://github.com/ExchangeUnion/xud/commit/ba041938cdcddb3aedec0db8fe4f571e2608b9d1))
+* **connext:** request collateral for order amount ([75078c0](https://github.com/ExchangeUnion/xud/commit/75078c059bfd752ae47c4530a671bd7a95568975)), closes [#1845](https://github.com/ExchangeUnion/xud/issues/1845)
+* **rpc:** SetLogLevel ([#1955](https://github.com/ExchangeUnion/xud/issues/1955)) ([8d10df0](https://github.com/ExchangeUnion/xud/commit/8d10df00f7442ea58286726f0348d0f43eae7622)), closes [#835](https://github.com/ExchangeUnion/xud/issues/835)
+* added new grpc method to easily cancel all orders ([#1910](https://github.com/ExchangeUnion/xud/issues/1910)) ([c958e86](https://github.com/ExchangeUnion/xud/commit/c958e862542fb52fdeb6532cbe2d92dd4b52cedf))
+* **cli:** case insensitive choices ([4c01b0e](https://github.com/ExchangeUnion/xud/commit/4c01b0e5fe5da75bcb3b4de063e0132929b358bb))
```
Comment (kilrau) [inline CHANGELOG.md:52]:
```suggestion

```

File: CHANGELOG.md [hunk 18]
```diff
@@ -1,3 +1,74 @@
...
+* **connext:** lazy collateralization ([#1916](https://github.com/ExchangeUnion/xud/issues/1916)) ([0f2b841](https://github.com/ExchangeUnion/xud/commit/0f2b841d3bc353cebca01f0bcdfec52c6c13e9d0)), closes [#1896](https://github.com/ExchangeUnion/xud/issues/1896)
```
Comment (kilrau) [inline CHANGELOG.md:53]:
```suggestion

```

File: CHANGELOG.md [hunk 19]
```diff
@@ -1,3 +1,74 @@
...
+* reserved capacity checks on PlaceOrder ([#1949](https://github.com/ExchangeUnion/xud/issues/1949)) ([d458745](https://github.com/ExchangeUnion/xud/commit/d4587458ab15a43bb6f4cfea18ee13cbd2653075)), closes [#1947](https://github.com/ExchangeUnion/xud/issues/1947)
+* **lnd:** change gRPC client options ([aefdce8](https://github.com/ExchangeUnion/xud/commit/aefdce84e2c6cae35bf243f21a5224427fe2bfcc))
+* derive child seed for swap clients ([be4121f](https://github.com/ExchangeUnion/xud/commit/be4121fd04156318c68dba28ee5f1a2483fd0c07)), closes [#1576](https://github.com/ExchangeUnion/xud/issues/1576)
+* removeorder output changed to a more meaningful message ([#1526](https://github.com/ExchangeUnion/xud/issues/1526)) ([ba04193](https://github.com/ExchangeUnion/xud/commit/ba041938cdcddb3aedec0db8fe4f571e2608b9d1))
+* **connext:** request collateral for order amount ([75078c0](https://github.com/ExchangeUnion/xud/commit/75078c059bfd752ae47c4530a671bd7a95568975)), closes [#1845](https://github.com/ExchangeUnion/xud/issues/1845)
```
Comment (kilrau) [inline CHANGELOG.md:49]:
```suggestion

```

File: CHANGELOG.md [hunk 20]
```diff
@@ -1,3 +1,74 @@
...
+* **cli:** case insensitive choices ([4c01b0e](https://github.com/ExchangeUnion/xud/commit/4c01b0e5fe5da75bcb3b4de063e0132929b358bb))
+* **connext:** lazy collateralization ([#1916](https://github.com/ExchangeUnion/xud/issues/1916)) ([0f2b841](https://github.com/ExchangeUnion/xud/commit/0f2b841d3bc353cebca01f0bcdfec52c6c13e9d0)), closes [#1896](https://github.com/ExchangeUnion/xud/issues/1896)
+* **connext:** reject app install for transfers without status field ([#1863](https://github.com/ExchangeUnion/xud/issues/1863)) ([519aa54](https://github.com/ExchangeUnion/xud/commit/519aa54fa9bfd138d6086f0de0406462a62a57bc))
+* **connext:** unlock expired transfer apps ([#1857](https://github.com/ExchangeUnion/xud/issues/1857)) ([023434d](https://github.com/ExchangeUnion/xud/commit/023434de1eabe4811986b9156beef7a5a670c047))
+* **logging:** order holds on trace level ([#1865](https://github.com/ExchangeUnion/xud/issues/1865)) ([5e3ad04](https://github.com/ExchangeUnion/xud/commit/5e3ad04ed8f2e35cb06479d2172d25bffa002768))
```
Comment (kilrau) [inline CHANGELOG.md:56]:
```suggestion

```

Comment (kilrau) [inline CHANGELOG.md:56]:
```suggestion

```

File: CHANGELOG.md [hunk 21]
```diff
@@ -1,3 +1,74 @@
...
+* **orderbook:** better replace order hold message ([b7fa00a](https://github.com/ExchangeUnion/xud/commit/b7fa00a7afc8ab0fe4e4f495395434e49e00f357))
+* **orderbook:** log error message on remove order ([#1901](https://github.com/ExchangeUnion/xud/issues/1901)) ([788490f](https://github.com/ExchangeUnion/xud/commit/788490f430691e2e93a28e4aab39aa260213e5b3))
+* **p2p:** don't log empty order packets ([#1871](https://github.com/ExchangeUnion/xud/issues/1871)) ([1b6d6d6](https://github.com/ExchangeUnion/xud/commit/1b6d6d6cf522517dbe138b8c7dda908b36f2d1a4))
+* **p2p:** increase reconnection delay backoff ([#1870](https://github.com/ExchangeUnion/xud/issues/1870)) ([44af197](https://github.com/ExchangeUnion/xud/commit/44af19722ced03fa2fd89702fea88d07ef51ae08))
+* **rpc:** add txid to open/close channel response ([0669a3f](https://github.com/ExchangeUnion/xud/commit/0669a3f41f6a8de9cc6afcf4d8d58c91d18d5b58)), closes [#1860](https://github.com/ExchangeUnion/xud/issues/1860)
+* **rpc:** expose reserved balance for GetBalance ([#1925](https://github.com/ExchangeUnion/xud/issues/1925)) ([8b18dd7](https://github.com/ExchangeUnion/xud/commit/8b18dd7426f01a760e571a0a5bb2310759229b64))
```
Comment (kilrau) [inline CHANGELOG.md:62]:
```suggestion

```

File: CHANGELOG.md [hunk 22]
```diff
@@ -1,3 +1,74 @@
...
+* **rpc:** reserved order amount for TradingLimits ([7f7cd68](https://github.com/ExchangeUnion/xud/commit/7f7cd683272e738a77480028c16095e8fb34c175)), closes [#1584](https://github.com/ExchangeUnion/xud/issues/1584) [#1678](https://github.com/ExchangeUnion/xud/issues/1678)
```
Comment (kilrau) [inline CHANGELOG.md:63]:
```suggestion

```

File: CHANGELOG.md [hunk 23]
```diff
@@ -1,3 +1,74 @@
...
+* **rpc:** runtime addcurrency for lnd & connext ([#1746](https://github.com/ExchangeUnion/xud/issues/1746)) ([fc83823](https://github.com/ExchangeUnion/xud/commit/fc8382313854b259ca97256004c8ddbb35dcffca))
+* **swapclient:** auto init wallets on xud unlock ([#1973](https://github.com/ExchangeUnion/xud/issues/1973)) ([a1287dd](https://github.com/ExchangeUnion/xud/commit/a1287dda7740a92e92d8799bf4f2a8a0279d21a8)), closes [#1929](https://github.com/ExchangeUnion/xud/issues/1929)
+* reserved capacity checks on PlaceOrder ([#1949](https://github.com/ExchangeUnion/xud/issues/1949)) ([d458745](https://github.com/ExchangeUnion/xud/commit/d4587458ab15a43bb6f4cfea18ee13cbd2653075)), closes [#1947](https://github.com/ExchangeUnion/xud/issues/1947)
+* **lnd:** change gRPC client options ([aefdce8](https://github.com/ExchangeUnion/xud/commit/aefdce84e2c6cae35bf243f21a5224427fe2bfcc))
+* derive child seed for swap clients ([be4121f](https://github.com/ExchangeUnion/xud/commit/be4121fd04156318c68dba28ee5f1a2483fd0c07)), closes [#1576](https://github.com/ExchangeUnion/xud/issues/1576)
```
Comment (kilrau) [inline CHANGELOG.md:47]:
```suggestion

```

File: CHANGELOG.md [hunk 24]
```diff
@@ -1,3 +1,74 @@
...
+* **rpc:** SetLogLevel ([#1955](https://github.com/ExchangeUnion/xud/issues/1955)) ([8d10df0](https://github.com/ExchangeUnion/xud/commit/8d10df00f7442ea58286726f0348d0f43eae7622)), closes [#835](https://github.com/ExchangeUnion/xud/issues/835)
+* added new grpc method to easily cancel all orders ([#1910](https://github.com/ExchangeUnion/xud/issues/1910)) ([c958e86](https://github.com/ExchangeUnion/xud/commit/c958e862542fb52fdeb6532cbe2d92dd4b52cedf))
+* **cli:** case insensitive choices ([4c01b0e](https://github.com/ExchangeUnion/xud/commit/4c01b0e5fe5da75bcb3b4de063e0132929b358bb))
+* **connext:** lazy collateralization ([#1916](https://github.com/ExchangeUnion/xud/issues/1916)) ([0f2b841](https://github.com/ExchangeUnion/xud/commit/0f2b841d3bc353cebca01f0bcdfec52c6c13e9d0)), closes [#1896](https://github.com/ExchangeUnion/xud/issues/1896)
+* **connext:** reject app install for transfers without status field ([#1863](https://github.com/ExchangeUnion/xud/issues/1863)) ([519aa54](https://github.com/ExchangeUnion/xud/commit/519aa54fa9bfd138d6086f0de0406462a62a57bc))
```
Comment (kilrau) [inline CHANGELOG.md:54]:
```suggestion

```

File: CHANGELOG.md [hunk 25]
```diff
@@ -1,3 +1,74 @@
...
+* **connext:** unlock expired transfer apps ([#1857](https://github.com/ExchangeUnion/xud/issues/1857)) ([023434d](https://github.com/ExchangeUnion/xud/commit/023434de1eabe4811986b9156beef7a5a670c047))
```
Comment (kilrau) [inline CHANGELOG.md:55]:
```suggestion

```

File: CHANGELOG.md [hunk 26]
```diff
@@ -1,3 +1,74 @@
...
+* **p2p:** increase reconnection delay backoff ([#1870](https://github.com/ExchangeUnion/xud/issues/1870)) ([44af197](https://github.com/ExchangeUnion/xud/commit/44af19722ced03fa2fd89702fea88d07ef51ae08))
+* **rpc:** add txid to open/close channel response ([0669a3f](https://github.com/ExchangeUnion/xud/commit/0669a3f41f6a8de9cc6afcf4d8d58c91d18d5b58)), closes [#1860](https://github.com/ExchangeUnion/xud/issues/1860)
+* **rpc:** expose reserved balance for GetBalance ([#1925](https://github.com/ExchangeUnion/xud/issues/1925)) ([8b18dd7](https://github.com/ExchangeUnion/xud/commit/8b18dd7426f01a760e571a0a5bb2310759229b64))
+* **rpc:** reserved order amount for TradingLimits ([7f7cd68](https://github.com/ExchangeUnion/xud/commit/7f7cd683272e738a77480028c16095e8fb34c175)), closes [#1584](https://github.com/ExchangeUnion/xud/issues/1584) [#1678](https://github.com/ExchangeUnion/xud/issues/1678)
+* **rpc:** show connext status on create/restore node ([#1902](https://github.com/ExchangeUnion/xud/issues/1902)) ([82e16a5](https://github.com/ExchangeUnion/xud/commit/82e16a5e4ed4fa91e481eac638b16eb10e5bb7de))
```
Comment (kilrau) [inline CHANGELOG.md:64]:
```suggestion

```

File: CHANGELOG.md [hunk 27]
```diff
@@ -1,3 +1,74 @@
...
+* **simnet:** add DAI ([#1915](https://github.com/ExchangeUnion/xud/issues/1915)) ([35e85b3](https://github.com/ExchangeUnion/xud/commit/35e85b31bfbfa72430f5db982b7f64f19b346f2a))
```
Comment (kilrau) [inline CHANGELOG.md:65]:
```suggestion

```

File: CHANGELOG.md [hunk 28]
```diff
@@ -1,3 +1,74 @@
...
+* sat_per_byte custom fee for openchannel ([#1832](https://github.com/ExchangeUnion/xud/issues/1832)) ([b56ad98](https://github.com/ExchangeUnion/xud/commit/b56ad98ecb31f94d304a31d3c060693506ae829a)), closes [#1829](https://github.com/ExchangeUnion/xud/issues/1829)
```
Comment (kilrau) [inline CHANGELOG.md:66]:
```suggestion

```

File: CHANGELOG.md [hunk 29]
```diff
@@ -1,3 +1,74 @@
...
+* **cli:** case insensitive choices ([4c01b0e](https://github.com/ExchangeUnion/xud/commit/4c01b0e5fe5da75bcb3b4de063e0132929b358bb))
+* **connext:** lazy collateralization ([#1916](https://github.com/ExchangeUnion/xud/issues/1916)) ([0f2b841](https://github.com/ExchangeUnion/xud/commit/0f2b841d3bc353cebca01f0bcdfec52c6c13e9d0)), closes [#1896](https://github.com/ExchangeUnion/xud/issues/1896)
+* **connext:** reject app install for transfers without status field ([#1863](https://github.com/ExchangeUnion/xud/issues/1863)) ([519aa54](https://github.com/ExchangeUnion/xud/commit/519aa54fa9bfd138d6086f0de0406462a62a57bc))
+* **connext:** unlock expired transfer apps ([#1857](https://github.com/ExchangeUnion/xud/issues/1857)) ([023434d](https://github.com/ExchangeUnion/xud/commit/023434de1eabe4811986b9156beef7a5a670c047))
+* **logging:** order holds on trace level ([#1865](https://github.com/ExchangeUnion/xud/issues/1865)) ([5e3ad04](https://github.com/ExchangeUnion/xud/commit/5e3ad04ed8f2e35cb06479d2172d25bffa002768))
+* **orderbook:** better replace order hold message ([b7fa00a](https://github.com/ExchangeUnion/xud/commit/b7fa00a7afc8ab0fe4e4f495395434e49e00f357))
```
Comment (kilrau) [inline CHANGELOG.md:57]:
```suggestion

```

File: CHANGELOG.md [hunk 30]
```diff
@@ -1,3 +1,74 @@
...
+* **orderbook:** log error message on remove order ([#1901](https://github.com/ExchangeUnion/xud/issues/1901)) ([788490f](https://github.com/ExchangeUnion/xud/commit/788490f430691e2e93a28e4aab39aa260213e5b3))
```
Comment (kilrau) [inline CHANGELOG.md:58]:
```suggestion

```

File: CHANGELOG.md [hunk 31]
```diff
@@ -1,3 +1,74 @@
...
+* **p2p:** don't log empty order packets ([#1871](https://github.com/ExchangeUnion/xud/issues/1871)) ([1b6d6d6](https://github.com/ExchangeUnion/xud/commit/1b6d6d6cf522517dbe138b8c7dda908b36f2d1a4))
```
Comment (kilrau) [inline CHANGELOG.md:59]:
```suggestion

```

File: CHANGELOG.md [hunk 32]
```diff
@@ -1,3 +1,74 @@
...
+* **p2p:** increase reconnection delay backoff ([#1870](https://github.com/ExchangeUnion/xud/issues/1870)) ([44af197](https://github.com/ExchangeUnion/xud/commit/44af19722ced03fa2fd89702fea88d07ef51ae08))
```
Comment (kilrau) [inline CHANGELOG.md:60]:
```suggestion

```

File: CHANGELOG.md [hunk 33]
```diff
@@ -1,3 +1,74 @@
...
+* **rpc:** reserved order amount for TradingLimits ([7f7cd68](https://github.com/ExchangeUnion/xud/commit/7f7cd683272e738a77480028c16095e8fb34c175)), closes [#1584](https://github.com/ExchangeUnion/xud/issues/1584) [#1678](https://github.com/ExchangeUnion/xud/issues/1678)
+* **rpc:** show connext status on create/restore node ([#1902](https://github.com/ExchangeUnion/xud/issues/1902)) ([82e16a5](https://github.com/ExchangeUnion/xud/commit/82e16a5e4ed4fa91e481eac638b16eb10e5bb7de))
+* **simnet:** add DAI ([#1915](https://github.com/ExchangeUnion/xud/issues/1915)) ([35e85b3](https://github.com/ExchangeUnion/xud/commit/35e85b31bfbfa72430f5db982b7f64f19b346f2a))
+* sat_per_byte custom fee for openchannel ([#1832](https://github.com/ExchangeUnion/xud/issues/1832)) ([b56ad98](https://github.com/ExchangeUnion/xud/commit/b56ad98ecb31f94d304a31d3c060693506ae829a)), closes [#1829](https://github.com/ExchangeUnion/xud/issues/1829)
+* **simnet:** change USDT contract address ([#1912](https://github.com/ExchangeUnion/xud/issues/1912)) ([0a2bdfd](https://github.com/ExchangeUnion/xud/commit/0a2bdfdba3507cbd0bba1a0af6d6f8b8b92c3272))
```
Comment (kilrau) [inline CHANGELOG.md:67]:
```suggestion

```

File: CHANGELOG.md [hunk 34]
```diff
@@ -1,3 +1,74 @@
...
+* **simnet:** upgrade simnet USDT contract address ([#1906](https://github.com/ExchangeUnion/xud/issues/1906)) ([7e5f8ef](https://github.com/ExchangeUnion/xud/commit/7e5f8ef30a1d01a4405757d4359b827ef86cf750))
```
Comment (kilrau) [inline CHANGELOG.md:68]:
```suggestion

```

**Review by kilrau — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 35]
```diff
@@ -1,3 +1,41 @@
...
+* **lnd:** change gRPC client options ([aefdce8](https://github.com/ExchangeUnion/xud/commit/aefdce84e2c6cae35bf243f21a5224427fe2bfcc))
+* removeorder output changed to a more meaningful message ([#1526](https://github.com/ExchangeUnion/xud/issues/1526)) ([ba04193](https://github.com/ExchangeUnion/xud/commit/ba041938cdcddb3aedec0db8fe4f571e2608b9d1))
+* **rpc:** SetLogLevel ([#1955](https://github.com/ExchangeUnion/xud/issues/1955)) ([8d10df0](https://github.com/ExchangeUnion/xud/commit/8d10df00f7442ea58286726f0348d0f43eae7622)), closes [#835](https://github.com/ExchangeUnion/xud/issues/835)
+* added new grpc method to easily cancel all orders ([#1910](https://github.com/ExchangeUnion/xud/issues/1910)) ([c958e86](https://github.com/ExchangeUnion/xud/commit/c958e862542fb52fdeb6532cbe2d92dd4b52cedf))
+* **rpc:** add txid to open/close channel response ([0669a3f](https://github.com/ExchangeUnion/xud/commit/0669a3f41f6a8de9cc6afcf4d8d58c91d18d5b58)), closes [#1860](https://github.com/ExchangeUnion/xud/issues/1860)
```
Comment (kilrau) [inline CHANGELOG.md:35]:
```suggestion

```

**Review by kilrau — CHANGES_REQUESTED** [review]

**Review by kilrau — APPROVED** [review]

**Pull request merged**